### ISSUE TITLE
Did the default faster rcnn configure got learning rate decay?

### ISSUE THREAD

**Issue by Jack12xl** [open-mmlab/mmdetection#2031]
In the default faster rcnn [configure](https://github.com/open-mmlab/mmdetection/blob/aade6801e7df66679b1fe9d162da0d03b4742dd4/configs/faster_rcnn_r101_fpn_1x.py#L152), we have 

`
lr_config = dict(
    policy='step',
    warmup='linear',
    warmup_iters=500,
    warmup_ratio=1.0 / 3,
    step=[8, 11])
`

It seems it does not explicitly contain a parameter to tune the learning rate decay. 
As I noticed the related topic in #1430, 

> the learning rate will decay to 0.1*itself at the specific epoch

It implies the default configure did have lr decay feature. So I wonder where and how the code implement the lr decay ?

**Comment by Jack12xl** [comment]
in [apis/train.py](https://github.com/open-mmlab/mmdetection/blob/4b984a7c543e7dfbeafe613ff0db18a7fdb76f43/mmdet/apis/train.py#L145)  the function build_optimizer has the parameter-wise LR decay parameter. 

> `bias_lr_mult` and `bias_decay_mult` will be multiplied to
                  the lr and weight decay respectively for all bias parameters

But when I train the default configure,  it did not contain these parameter
<img width="633" alt="image" src="https://user-images.githubusercontent.com/27931931/73516019-36671400-4432-11ea-8e7c-6b0f83be712e.png">

**Comment by hellock** [comment]
The default gamma (decay ratio) is 0.1, see [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py#L102).

You may understood the parameter `bias_lr_mult` and `bias_decay_mult`, which are explained in the documentation. By default they are not used.
